src/targeting/Semgrepignore.ml

```diff
@@ -106,7 +106,7 @@ let contents_of_builtin_semgrepignore = function
   | Empty -> ""
   | Semgrep_scan_legacy -> default_semgrepignore_for_semgrep_scan
 
-let create ?(cli_patterns = []) ~default_semgrepignore_patterns
+let create ?(cli_patterns = []) ?(semgrepignore_filename = None) ~default_semgrepignore_patterns
```

Comment:
This new parameter should be:

```ocaml
?(semgrepignore_filename = default_semgrepignore_filename)
```

where `default_semgrepignore_filename` should be defined near the top of the file, to be equal to `".semgrepignore"`.

As a side-note, if you wanted `None` for missing, you could just write `?semgrepignore_filename`.